<launch>
  
  <node pkg="inverse_kinetic" type="inverse_kinetic" name="inverse_kinetic" required="true" output = "screen" > 
    <param name="topic_velocities" value="topic_velocities"/>
    <param name="cmd_vel" value="cmd_vel"/>
    <param name="left_wheel_velocity" type="int" value="0" />
    <param name="right_wheel_velocity" type="int" value="0" />
    <param name="v_max_motor" type="int" value="14000" />
    <param name="pwm_max" type="int" value="255" />
    <param name="steering_factor" type="double" value="19.2" />
    <param name="linear_vel" type="double" value="0" />
    <param name="angular_vel" type="double" value="0" />
    <param name="wheels_distance" type="double" value="0.45" />
    <param name="t_acceleration" type="double" value="1" />
    <param name="t_decreasing" type="double" value="0.5" />
  </node>

</launch>
